_posts/2020/11/2020-11-23-python-repository.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+date: 2020-11-23
+title: "Python And Binary Repositories"
+author: olenagerasimova
+description: This blog post shows how binary and pypi repositories can be used.
+keywords:
+  - artipie
+  - private
+  - pypi
+  - python
+  - repository
+  - registry
+  - binary
+---
+
+Being a student I dreamed about something like Artipie, I needed some private centralized storage 
+where I could have kept both binary or text data files and Python packages. We were conducting a research
+and used to work with a lot of data from experimental measurements and used Python to process and 
+visualise them. My university colleagues from that time were mathematicians, they did not have strong
+development background and were absolutely fine with passing data and code via flash drive or email.
+All my efforts to inculcate VCS (GitHub, for example, provides free Pro accounts for students 
+and professors) and some cloud storage usage were always postponed to the brighter future. I've 
+quit the postgraduate program and never saw that future to come.
+
+Now, I'd like to show how I could have used Artipie being that student. In the example I'll use 
+[Artipie Central](http://central.artipie.com/), but the same will work with an Artipie instance 
+running on your private server in the private network.
+
+## Python Repository
+
+Artipie supports [PyPi repository](https://github.com/artipie/artipie/tree/master/examples/pypi) which
+`twine` and `pip` can perfectly understand. This means that you can work with Artipie Python 
```

Comment:
@olenagerasimova I would add links to twine and pip. In general, it's good to have links to everything that you mention. This way you help the reader quickly find the right info, if he/she is interested.